src/color/background.json

```diff
@@ -151,15 +151,15 @@
       },
       "error": {
         "default": {
-          "value": "{color.base.error.100.value}",
+          "value": "{color.state.error.100.value}",
```

Comment:
According to this file, we should be using `base` not `state`:
- https://alaskaair-my.sharepoint.com/:x:/p/june_lee/EeyDi0RTK5xEosX7r0KZ_7MBIayy5sUdJ7WBT4XJZqHUmQ?e=ULdhyD&ovuser=0f44c5d4-42b0-45c2-bf55-d0fea8430d33%2CJordan.Jones4%40alaskaair.com&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiI1MC8yNDA0MTEyMjMxMyIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D